framework/doc/content/source/utils/BulkMaterial.md

```diff
@@ -0,0 +1,12 @@
+# BulkMaterial
```

Comment:
Don't forget to rename this file. Also, it should be in `source/userobject(s)`, right?